code/modules/projectiles/ammunition.dm

```diff
@@ -76,6 +77,13 @@
 				to_chat(user, "<span class='notice'>There is no bullet in the casing to inscribe anything into.</span>")
 		..()
 
+/obj/item/ammo_casing/proc/play_casing_sound()
+	//Used to play sound of spent casing hitting floor. Checks for gravity first.
+	spawn(5) //Gravity in effect, boys.

```

Comment:
I think we're trying to use `addtimer` for things like this rather than spawn now.

Reply:
@Tayyyyyyy is correct, and may I also suggest using a variable timer. Possibly a 5-10 deciseconds? I think that may sound nicer on the ears than a set time for every casing ejected.

Reply:
How would I go about implementing addtimer? (and the 5-10)